test/integration/framework.go

```diff
@@ -57,13 +57,13 @@ func getFreshApiserverAndClient(t *testing.T) (servicecatalogclient.Interface, f
 	go func() {
 
 		options := &server.ServiceCatalogServerOptions{
-			StorageTypeString:       "etcd",
+			StorageTypeString:       storageTypeStr,
 			GenericServerRunOptions: genericserveroptions.NewServerRunOptions(),
 			SecureServingOptions:    secureServingOptions,
 			EtcdOptions: &server.EtcdOptions{
 				EtcdOptions: genericserveroptions.NewEtcdOptions(),
 			},
-			TPROptions:            server.NewTPROptions(),
+			TPROptions:            &server.TPROptions{},
```

Comment:
no. we're gonna need a follow-up to enable actual TPR tests. In it, we'll be mocking out the `clientset.Interface` to return the "right stuff", so we won't need an API server